esp-wifi/src/ble/os_adapter_esp32.rs

```diff
@@ -285,12 +292,12 @@ static BTDM_DRAM_AVAILABLE_REGION: [btdm_dram_available_region_t; 7] = [
 pub(crate) fn create_ble_config() -> esp_bt_controller_config_t {
     esp_bt_controller_config_t {
         controller_task_stack_size: 4096,
-        controller_task_prio: 110,
+        controller_task_prio: 23,
```

Comment:
It would be very nice to have some explanation, or a cross-reference to where this comes from.

Reply:
This change is missing from the changelog, and its effects are unknown.

Reply:
we can revert the change to `controller_task_prio` - I doubt it will have any effect for our built-in-scheduler
was just aligning the settings when comparing things to esp-idf but none of the config changes should be needed for the fix